/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:283: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin torch._C._to_dlpack. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 02:34:04.108782 47994779129088 torch/_dynamo/guards.py:2611] [4/1] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:34:04.108782 47994779129088 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:04.108782 47994779129088 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:34:04.127100 47994779129088 torch/_dynamo/guards.py:2611] [5/1] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 02:34:04.127100 47994779129088 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:04.127100 47994779129088 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:34:04.138271 47994779129088 torch/_dynamo/guards.py:2611] [6/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:34:04.138271 47994779129088 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:04.138271 47994779129088 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:34:04.174337 47994779129088 torch/_dynamo/guards.py:2611] [8/1] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:34:04.174337 47994779129088 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:04.174337 47994779129088 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
W0421 02:34:10.990319 47994779129088 torch/_dynamo/variables/tensor.py:715] [30/0] Graph break from `Tensor.item()`, consider setting:
W0421 02:34:10.990319 47994779129088 torch/_dynamo/variables/tensor.py:715] [30/0]     torch._dynamo.config.capture_scalar_outputs = True
W0421 02:34:10.990319 47994779129088 torch/_dynamo/variables/tensor.py:715] [30/0] or:
W0421 02:34:10.990319 47994779129088 torch/_dynamo/variables/tensor.py:715] [30/0]     env TORCHDYNAMO_CAPTURE_SCALAR_OUTPUTS=1
W0421 02:34:10.990319 47994779129088 torch/_dynamo/variables/tensor.py:715] [30/0] to include these operations in the captured graph.
W0421 02:34:10.990319 47994779129088 torch/_dynamo/variables/tensor.py:715] [30/0] 
V0421 02:34:13.928995 47994779129088 torch/_dynamo/guards.py:2611] [4/2] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:34:13.928995 47994779129088 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:13.928995 47994779129088 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 845
V0421 02:34:13.928995 47994779129088 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:34:13.957537 47994779129088 torch/_dynamo/guards.py:2611] [5/2] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 02:34:13.957537 47994779129088 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:13.957537 47994779129088 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 845
V0421 02:34:13.957537 47994779129088 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:34:13.969374 47994779129088 torch/_dynamo/guards.py:2611] [6/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:34:13.969374 47994779129088 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:13.969374 47994779129088 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 845
V0421 02:34:13.969374 47994779129088 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:34:14.007133 47994779129088 torch/_dynamo/guards.py:2611] [8/2] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:34:14.007133 47994779129088 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:14.007133 47994779129088 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 845
V0421 02:34:14.007133 47994779129088 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:34:14.024355 47994779129088 torch/_dynamo/guards.py:2611] [32/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 02:34:14.024355 47994779129088 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:14.024355 47994779129088 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     - L['___stack0'] == 650                                       
V0421 02:34:14.255693 47994779129088 torch/_dynamo/guards.py:2611] [34/1] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 02:34:14.255693 47994779129088 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:14.255693 47994779129088 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 845
V0421 02:34:15.006698 47994779129088 torch/_dynamo/guards.py:2611] [20/1] [__recompiles] Recompiling function add_edges in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:435
V0421 02:34:15.006698 47994779129088 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:15.006698 47994779129088 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 845, actual 203
V0421 02:34:15.027708 47994779129088 torch/_dynamo/guards.py:2611] [26/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_541 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:541
V0421 02:34:15.027708 47994779129088 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:15.027708 47994779129088 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 845, actual 203
V0421 02:34:15.055766 47994779129088 torch/_dynamo/guards.py:2611] [27/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_542 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:542
V0421 02:34:15.055766 47994779129088 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:15.055766 47994779129088 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 845, actual 203
V0421 02:34:15.072077 47994779129088 torch/_dynamo/guards.py:2611] [28/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_545 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:545
V0421 02:34:15.072077 47994779129088 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:15.072077 47994779129088 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 845, actual 203
V0421 02:34:15.137016 47994779129088 torch/_dynamo/guards.py:2611] [30/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 02:34:15.137016 47994779129088 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:15.137016 47994779129088 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 845, actual 203
V0421 02:34:15.390212 47994779129088 torch/_dynamo/guards.py:2611] [32/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 02:34:15.390212 47994779129088 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:15.390212 47994779129088 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 845, actual 203
V0421 02:34:15.390212 47994779129088 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - L['___stack0'] == 650                                       
V0421 02:34:15.639059 47994779129088 torch/_dynamo/guards.py:2611] [34/2] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 02:34:15.639059 47994779129088 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:15.639059 47994779129088 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 845, actual 203
V0421 02:34:15.639059 47994779129088 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 1690
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin posix.stat. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 02:34:19.774475 47994779129088 torch/_dynamo/guards.py:2611] [6/3] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:34:19.774475 47994779129088 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:19.774475 47994779129088 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:34:19.774475 47994779129088 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:34:19.774475 47994779129088 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:34:19.820666 47994779129088 torch/_dynamo/guards.py:2611] [7/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_at_435 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:435
V0421 02:34:19.820666 47994779129088 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:19.820666 47994779129088 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin time.perf_counter. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 02:34:20.371633 47994779129088 torch/_dynamo/guards.py:2611] [86/1] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:34:20.371633 47994779129088 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:20.371633 47994779129088 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     - L['___stack0'] == 924301.843790886                          
V0421 02:34:20.463980 47994779129088 torch/_dynamo/guards.py:2611] [86/2] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:34:20.463980 47994779129088 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:20.463980 47994779129088 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 924301.856528954                          
V0421 02:34:20.463980 47994779129088 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 924301.843790886                          
V0421 02:34:23.481386 47994779129088 torch/_dynamo/guards.py:2611] [86/3] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:34:23.481386 47994779129088 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:23.481386 47994779129088 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 924301.948858909                          
V0421 02:34:23.481386 47994779129088 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 924301.856528954                          
V0421 02:34:23.481386 47994779129088 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 924301.843790886                          
V0421 02:34:25.486456 47994779129088 torch/_dynamo/guards.py:2611] [86/4] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:34:25.486456 47994779129088 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.486456 47994779129088 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 924304.966183632                          
V0421 02:34:25.486456 47994779129088 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 924301.948858909                          
V0421 02:34:25.486456 47994779129088 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 924301.856528954                          
V0421 02:34:25.486456 47994779129088 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 924301.843790886                          
V0421 02:34:25.556923 47994779129088 torch/_dynamo/guards.py:2611] [86/5] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:34:25.556923 47994779129088 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.556923 47994779129088 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 924306.971185004                          
V0421 02:34:25.556923 47994779129088 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 924304.966183632                          
V0421 02:34:25.556923 47994779129088 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 924301.948858909                          
V0421 02:34:25.556923 47994779129088 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 924301.856528954                          
V0421 02:34:25.556923 47994779129088 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 924301.843790886                          
V0421 02:34:25.563196 47994779129088 torch/_dynamo/guards.py:2611] [86/6] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:34:25.563196 47994779129088 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.563196 47994779129088 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 924307.04173942                           
V0421 02:34:25.563196 47994779129088 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 924306.971185004                          
V0421 02:34:25.563196 47994779129088 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 924304.966183632                          
V0421 02:34:25.563196 47994779129088 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 924301.948858909                          
V0421 02:34:25.563196 47994779129088 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 924301.856528954                          
V0421 02:34:25.563196 47994779129088 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 924301.843790886                          
V0421 02:34:25.621742 47994779129088 torch/_dynamo/guards.py:2611] [57/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 02:34:25.621742 47994779129088 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.621742 47994779129088 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:25.644504 47994779129088 torch/_dynamo/guards.py:2611] [61/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 02:34:25.644504 47994779129088 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.644504 47994779129088 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:25.664078 47994779129088 torch/_dynamo/guards.py:2611] [62/1] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 02:34:25.664078 47994779129088 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.664078 47994779129088 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:25.683928 47994779129088 torch/_dynamo/guards.py:2611] [63/1] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 02:34:25.683928 47994779129088 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.683928 47994779129088 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:25.701754 47994779129088 torch/_dynamo/guards.py:2611] [64/1] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 02:34:25.701754 47994779129088 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.701754 47994779129088 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:25.722409 47994779129088 torch/_dynamo/guards.py:2611] [65/1] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 02:34:25.722409 47994779129088 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.722409 47994779129088 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:25.768408 47994779129088 torch/_dynamo/guards.py:2611] [68/1] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 02:34:25.768408 47994779129088 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.768408 47994779129088 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:25.782124 47994779129088 torch/_dynamo/guards.py:2611] [70/1] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 02:34:25.782124 47994779129088 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.782124 47994779129088 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:25.800036 47994779129088 torch/_dynamo/guards.py:2611] [74/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 02:34:25.800036 47994779129088 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.800036 47994779129088 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:25.816365 47994779129088 torch/_dynamo/guards.py:2611] [77/1] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 02:34:25.816365 47994779129088 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:25.816365 47994779129088 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     - L['shape'][1] == 500                                        
V0421 02:34:26.086049 47994779129088 torch/_dynamo/guards.py:2611] [6/4] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:34:26.086049 47994779129088 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:26.086049 47994779129088 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:34:26.086049 47994779129088 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:26.086049 47994779129088 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:34:26.086049 47994779129088 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:34:26.141781 47994779129088 torch/_dynamo/guards.py:2611] [79/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 02:34:26.141781 47994779129088 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:26.141781 47994779129088 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:34:26.179037 47994779129088 torch/_dynamo/guards.py:2611] [80/1] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 02:34:26.179037 47994779129088 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:26.179037 47994779129088 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:34:26.189383 47994779129088 torch/_dynamo/guards.py:2611] [81/1] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 02:34:26.189383 47994779129088 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:26.189383 47994779129088 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:34:26.200997 47994779129088 torch/_dynamo/guards.py:2611] [82/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 02:34:26.200997 47994779129088 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:26.200997 47994779129088 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:34:26.214944 47994779129088 torch/_dynamo/guards.py:2611] [83/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_165 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:165
V0421 02:34:26.214944 47994779129088 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:26.214944 47994779129088 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:26.260465 47994779129088 torch/_dynamo/guards.py:2611] [86/7] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:34:26.260465 47994779129088 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:26.260465 47994779129088 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 924307.048042312                          
V0421 02:34:26.260465 47994779129088 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 924307.04173942                           
V0421 02:34:26.260465 47994779129088 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 924306.971185004                          
V0421 02:34:26.260465 47994779129088 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 924304.966183632                          
V0421 02:34:26.260465 47994779129088 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 924301.948858909                          
V0421 02:34:26.260465 47994779129088 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 924301.856528954                          
V0421 02:34:26.260465 47994779129088 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 924301.843790886                          
V0421 02:34:26.266782 47994779129088 torch/_dynamo/guards.py:2611] [86/8] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:34:26.266782 47994779129088 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:26.266782 47994779129088 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 924307.745245297                          
V0421 02:34:26.266782 47994779129088 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 924307.048042312                          
V0421 02:34:26.266782 47994779129088 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 924307.04173942                           
V0421 02:34:26.266782 47994779129088 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 924306.971185004                          
V0421 02:34:26.266782 47994779129088 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 924304.966183632                          
V0421 02:34:26.266782 47994779129088 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 924301.948858909                          
V0421 02:34:26.266782 47994779129088 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 924301.856528954                          
V0421 02:34:26.266782 47994779129088 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 924301.843790886                          
W0421 02:34:26.266925 47994779129088 torch/_dynamo/convert_frame.py:762] [86/8] torch._dynamo hit config.cache_size_limit (8)
W0421 02:34:26.266925 47994779129088 torch/_dynamo/convert_frame.py:762] [86/8]    function: 'torch_dynamo_resume_in_hook_at_12' (/work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12)
W0421 02:34:26.266925 47994779129088 torch/_dynamo/convert_frame.py:762] [86/8]    last reason: L['___stack0'] == 924301.843790886                          
W0421 02:34:26.266925 47994779129088 torch/_dynamo/convert_frame.py:762] [86/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 02:34:26.266925 47994779129088 torch/_dynamo/convert_frame.py:762] [86/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:34:27.517610 47994779129088 torch/_dynamo/guards.py:2611] [0/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:449
V0421 02:34:27.517610 47994779129088 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:27.517610 47994779129088 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:27.552784 47994779129088 torch/_dynamo/guards.py:2611] [1/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:154
V0421 02:34:27.552784 47994779129088 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:27.552784 47994779129088 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 651, actual 647
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:34:27.569477 47994779129088 torch/_dynamo/guards.py:2611] [9/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 02:34:27.569477 47994779129088 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:27.569477 47994779129088 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:27.584816 47994779129088 torch/_dynamo/guards.py:2611] [13/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 02:34:27.584816 47994779129088 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:27.584816 47994779129088 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:27.629285 47994779129088 torch/_dynamo/guards.py:2611] [4/3] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:34:27.629285 47994779129088 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:27.629285 47994779129088 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:34:27.629285 47994779129088 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 651                                         
V0421 02:34:27.629285 47994779129088 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 651                                         
V0421 02:34:27.661322 47994779129088 torch/_dynamo/guards.py:2611] [8/3] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:34:27.661322 47994779129088 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:27.661322 47994779129088 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:34:27.661322 47994779129088 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 651                                       
V0421 02:34:27.661322 47994779129088 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 651                                       
V0421 02:34:27.675700 47994779129088 torch/_dynamo/guards.py:2611] [15/1] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:34:27.675700 47994779129088 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:27.675700 47994779129088 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     - L['data'][0] == 651                                         
V0421 02:34:30.906750 47994779129088 torch/_dynamo/guards.py:2611] [29/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_572 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:572
V0421 02:34:30.906750 47994779129088 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:30.906750 47994779129088 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     - L['___stack0'] == 651                                       
V0421 02:34:30.924729 47994779129088 torch/_dynamo/guards.py:2611] [30/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 02:34:30.924729 47994779129088 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:30.924729 47994779129088 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - L['___stack0'] == 651                                       
V0421 02:34:30.924729 47994779129088 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 845, actual 823
V0421 02:34:31.179823 47994779129088 torch/_dynamo/guards.py:2611] [34/3] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 02:34:31.179823 47994779129088 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:31.179823 47994779129088 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - 2 <= L['seq'][0].size()[0] <= 2147483645                      # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:31.179823 47994779129088 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 845, actual 823
V0421 02:34:31.179823 47994779129088 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 845, actual 823
V0421 02:34:31.421137 47994779129088 torch/_dynamo/guards.py:2611] [4/4] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:34:31.421137 47994779129088 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:31.421137 47994779129088 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 823
V0421 02:34:31.421137 47994779129088 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:34:31.421137 47994779129088 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - L['num_dst'] == 651                                         
V0421 02:34:31.421137 47994779129088 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 823
V0421 02:34:31.460088 47994779129088 torch/_dynamo/guards.py:2611] [8/4] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:34:31.460088 47994779129088 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:31.460088 47994779129088 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 823
V0421 02:34:31.460088 47994779129088 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:34:31.460088 47994779129088 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - L['___stack2'] == 651                                       
V0421 02:34:31.460088 47994779129088 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 823
V0421 02:34:31.482331 47994779129088 torch/_dynamo/guards.py:2611] [41/1] [__recompiles] Recompiling function spmatrix in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/sparse/sparse_matrix.py:763
V0421 02:34:31.482331 47994779129088 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:31.482331 47994779129088 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     - tensor 'L['indices']' stride mismatch at index 0. expected 2096, actual 2052
V0421 02:34:31.756050 47994779129088 torch/_dynamo/guards.py:2611] [47/1] [__recompiles] Recompiling function arange in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:406
V0421 02:34:31.756050 47994779129088 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:31.756050 47994779129088 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     - L['stop'] == 651                                            
V0421 02:34:31.983293 47994779129088 torch/_dynamo/guards.py:2611] [48/1] [__recompiles] Recompiling function in_degrees in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:702
V0421 02:34:31.983293 47994779129088 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:31.983293 47994779129088 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.003135 47994779129088 torch/_dynamo/guards.py:2611] [50/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2878
V0421 02:34:32.003135 47994779129088 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.003135 47994779129088 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     - tensor 'L['vid']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.019528 47994779129088 torch/_dynamo/guards.py:2611] [51/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2925 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2925
V0421 02:34:32.019528 47994779129088 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.019528 47994779129088 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.261308 47994779129088 torch/_dynamo/guards.py:2611] [52/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2926 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2926
V0421 02:34:32.261308 47994779129088 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.261308 47994779129088 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     - tensor 'L['vid_tensor']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.285111 47994779129088 torch/_dynamo/guards.py:2611] [53/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:444
V0421 02:34:32.285111 47994779129088 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.285111 47994779129088 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     - tensor 'L['vids']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.302245 47994779129088 torch/_dynamo/guards.py:2611] [54/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2930 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2930
V0421 02:34:32.302245 47994779129088 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.302245 47994779129088 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.528217 47994779129088 torch/_dynamo/guards.py:2611] [55/1] [__recompiles] Recompiling function sum in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:152
V0421 02:34:32.528217 47994779129088 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.528217 47994779129088 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.608823 47994779129088 torch/_dynamo/guards.py:2611] [57/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 02:34:32.608823 47994779129088 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.608823 47994779129088 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=1
V0421 02:34:32.608823 47994779129088 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.627624 47994779129088 torch/_dynamo/guards.py:2611] [61/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 02:34:32.627624 47994779129088 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.627624 47994779129088 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 02:34:32.627624 47994779129088 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.646497 47994779129088 torch/_dynamo/guards.py:2611] [62/2] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 02:34:32.646497 47994779129088 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.646497 47994779129088 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 02:34:32.646497 47994779129088 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.666899 47994779129088 torch/_dynamo/guards.py:2611] [63/2] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 02:34:32.666899 47994779129088 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.666899 47994779129088 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 02:34:32.666899 47994779129088 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.684111 47994779129088 torch/_dynamo/guards.py:2611] [64/2] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 02:34:32.684111 47994779129088 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.684111 47994779129088 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=1
V0421 02:34:32.684111 47994779129088 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.703469 47994779129088 torch/_dynamo/guards.py:2611] [65/2] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 02:34:32.703469 47994779129088 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.703469 47994779129088 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=1
V0421 02:34:32.703469 47994779129088 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.748301 47994779129088 torch/_dynamo/guards.py:2611] [68/2] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 02:34:32.748301 47994779129088 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.748301 47994779129088 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=1
V0421 02:34:32.748301 47994779129088 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.762160 47994779129088 torch/_dynamo/guards.py:2611] [70/2] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 02:34:32.762160 47994779129088 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.762160 47994779129088 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=1
V0421 02:34:32.762160 47994779129088 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.780262 47994779129088 torch/_dynamo/guards.py:2611] [74/2] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 02:34:32.780262 47994779129088 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.780262 47994779129088 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=1
V0421 02:34:32.780262 47994779129088 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.796716 47994779129088 torch/_dynamo/guards.py:2611] [77/2] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 02:34:32.796716 47994779129088 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.796716 47994779129088 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 651                                        
V0421 02:34:32.796716 47994779129088 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 651                                        
V0421 02:34:32.967279 47994779129088 torch/_dynamo/guards.py:2611] [79/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 02:34:32.967279 47994779129088 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:32.967279 47994779129088 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:32.967279 47994779129088 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:33.010022 47994779129088 torch/_dynamo/guards.py:2611] [80/2] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 02:34:33.010022 47994779129088 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:33.010022 47994779129088 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:33.010022 47994779129088 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:33.022219 47994779129088 torch/_dynamo/guards.py:2611] [81/2] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 02:34:33.022219 47994779129088 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:33.022219 47994779129088 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:33.022219 47994779129088 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:33.035815 47994779129088 torch/_dynamo/guards.py:2611] [82/2] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 02:34:33.035815 47994779129088 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:33.035815 47994779129088 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:33.035815 47994779129088 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:33.054587 47994779129088 torch/_dynamo/guards.py:2611] [88/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:569
V0421 02:34:33.054587 47994779129088 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:33.054587 47994779129088 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:33.148646 47994779129088 torch/_dynamo/guards.py:2611] [89/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_590 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:590
V0421 02:34:33.148646 47994779129088 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:33.148646 47994779129088 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:33.220322 47994779129088 torch/_dynamo/guards.py:2611] [90/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418
V0421 02:34:33.220322 47994779129088 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:33.220322 47994779129088 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:34.002459 47994779129088 torch/_dynamo/guards.py:2611] [93/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_432 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:432
V0421 02:34:34.002459 47994779129088 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:34.002459 47994779129088 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     - L['shape'][1] == 651                                        
V0421 02:34:35.329627 47994779129088 torch/_dynamo/guards.py:2611] [94/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_595 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:595
V0421 02:34:35.329627 47994779129088 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:35.329627 47994779129088 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:35.610687 47994779129088 torch/_dynamo/guards.py:2611] [95/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_604 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:604
V0421 02:34:35.610687 47994779129088 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:35.610687 47994779129088 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:36.358570 47994779129088 torch/_dynamo/guards.py:2611] [96/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_648 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:648
V0421 02:34:36.358570 47994779129088 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:36.358570 47994779129088 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:36.467977 47994779129088 torch/_dynamo/guards.py:2611] [57/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 02:34:36.467977 47994779129088 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:36.467977 47994779129088 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.467977 47994779129088 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:36.467977 47994779129088 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.489138 47994779129088 torch/_dynamo/guards.py:2611] [61/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 02:34:36.489138 47994779129088 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:36.489138 47994779129088 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.489138 47994779129088 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:36.489138 47994779129088 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.510391 47994779129088 torch/_dynamo/guards.py:2611] [62/3] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 02:34:36.510391 47994779129088 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:36.510391 47994779129088 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.510391 47994779129088 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:36.510391 47994779129088 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.532003 47994779129088 torch/_dynamo/guards.py:2611] [63/3] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 02:34:36.532003 47994779129088 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:36.532003 47994779129088 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.532003 47994779129088 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:36.532003 47994779129088 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.551513 47994779129088 torch/_dynamo/guards.py:2611] [64/3] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 02:34:36.551513 47994779129088 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:36.551513 47994779129088 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.551513 47994779129088 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:36.551513 47994779129088 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.573157 47994779129088 torch/_dynamo/guards.py:2611] [65/3] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 02:34:36.573157 47994779129088 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:36.573157 47994779129088 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.573157 47994779129088 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:36.573157 47994779129088 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.623356 47994779129088 torch/_dynamo/guards.py:2611] [68/3] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 02:34:36.623356 47994779129088 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:36.623356 47994779129088 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.623356 47994779129088 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:36.623356 47994779129088 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.638345 47994779129088 torch/_dynamo/guards.py:2611] [70/3] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 02:34:36.638345 47994779129088 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:36.638345 47994779129088 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.638345 47994779129088 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:36.638345 47994779129088 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.657410 47994779129088 torch/_dynamo/guards.py:2611] [74/3] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 02:34:36.657410 47994779129088 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:36.657410 47994779129088 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 02:34:36.657410 47994779129088 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 651, actual 647
V0421 02:34:36.657410 47994779129088 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:34:37.124483 47994779129088 torch/_dynamo/guards.py:2611] [15/2] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:34:37.124483 47994779129088 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:37.124483 47994779129088 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 647                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:37.124483 47994779129088 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 651                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:34:40.822576 47994779129088 torch/_dynamo/guards.py:2611] [15/3] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:34:40.822576 47994779129088 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:40.822576 47994779129088 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 665                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:40.822576 47994779129088 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 647                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:40.822576 47994779129088 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 651                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:34:44.885051 47994779129088 torch/_dynamo/guards.py:2611] [15/4] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:34:44.885051 47994779129088 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:44.885051 47994779129088 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 631                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:44.885051 47994779129088 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 665                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:44.885051 47994779129088 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 647                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:44.885051 47994779129088 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 651                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:34:48.599161 47994779129088 torch/_dynamo/guards.py:2611] [15/5] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:34:48.599161 47994779129088 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:48.599161 47994779129088 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 689                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:48.599161 47994779129088 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 631                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:48.599161 47994779129088 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 665                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:48.599161 47994779129088 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 647                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:48.599161 47994779129088 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 651                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:34:52.305998 47994779129088 torch/_dynamo/guards.py:2611] [15/6] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:34:52.305998 47994779129088 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:52.305998 47994779129088 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 641                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:52.305998 47994779129088 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 689                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:52.305998 47994779129088 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 631                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:52.305998 47994779129088 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 665                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:52.305998 47994779129088 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 647                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:52.305998 47994779129088 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 651                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:34:56.006356 47994779129088 torch/_dynamo/guards.py:2611] [15/7] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:34:56.006356 47994779129088 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:56.006356 47994779129088 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 656                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:56.006356 47994779129088 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 641                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:56.006356 47994779129088 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 689                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:56.006356 47994779129088 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 631                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:56.006356 47994779129088 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 665                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:56.006356 47994779129088 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 647                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:56.006356 47994779129088 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 651                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:34:59.734125 47994779129088 torch/_dynamo/guards.py:2611] [15/8] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:34:59.734125 47994779129088 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     triggered by the following guard failure(s):
V0421 02:34:59.734125 47994779129088 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 675                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:59.734125 47994779129088 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 656                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:59.734125 47994779129088 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 641                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:59.734125 47994779129088 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 689                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:59.734125 47994779129088 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 631                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:59.734125 47994779129088 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 665                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:59.734125 47994779129088 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 647                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:34:59.734125 47994779129088 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 651                                         
W0421 02:34:59.734352 47994779129088 torch/_dynamo/convert_frame.py:762] [15/8] torch._dynamo hit config.cache_size_limit (8)
W0421 02:34:59.734352 47994779129088 torch/_dynamo/convert_frame.py:762] [15/8]    function: '_dispatch' (/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65)
W0421 02:34:59.734352 47994779129088 torch/_dynamo/convert_frame.py:762] [15/8]    last reason: L['data'][0] == 651                                         
W0421 02:34:59.734352 47994779129088 torch/_dynamo/convert_frame.py:762] [15/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 02:34:59.734352 47994779129088 torch/_dynamo/convert_frame.py:762] [15/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
Compiling model with torch.compile()...
Saved sparsity snapshot to outputs_compiled/tran/pubmed/SAGE/seed_2/sparsity_before.csv
                               layer_name  num_elements  sparsity
0  _orig_mod.encoder.graph_layer_1.weight        250000       0.0
1    _orig_mod.encoder.graph_layer_1.bias           500     100.0
2  _orig_mod.encoder.graph_layer_2.weight         64000       0.0
3    _orig_mod.encoder.graph_layer_2.bias           128     100.0
4      _orig_mod.encoder.decoder_1.weight        250000       0.0
5        _orig_mod.encoder.decoder_1.bias           500       0.0
6      _orig_mod.encoder.decoder_2.weight        250000       0.0
7        _orig_mod.encoder.decoder_2.bias           500       0.0
8         _orig_mod.encoder.linear.weight           384       0.0
9           _orig_mod.encoder.linear.bias             3       0.0
Saved sparsity snapshot to outputs_compiled/tran/pubmed/SAGE/seed_2/sparsity_after.csv
                               layer_name  num_elements   sparsity
0  _orig_mod.encoder.graph_layer_1.weight        250000   1.738800
1    _orig_mod.encoder.graph_layer_1.bias           500   0.000000
2  _orig_mod.encoder.graph_layer_2.weight         64000   0.001563
3    _orig_mod.encoder.graph_layer_2.bias           128   0.000000
4      _orig_mod.encoder.decoder_1.weight        250000   0.000000
5        _orig_mod.encoder.decoder_1.bias           500   0.000000
6      _orig_mod.encoder.decoder_2.weight        250000  17.955200
7        _orig_mod.encoder.decoder_2.bias           500  14.200000
8         _orig_mod.encoder.linear.weight           384   0.000000
9           _orig_mod.encoder.linear.bias             3   0.000000
Saved forward timing report to outputs_compiled/tran/pubmed/SAGE/seed_2/forward_times.csv
                         layer_name  avg_forward_time_ms  num_calls
0         _orig_mod.encoder.dropout             0.000522      20039
1   _orig_mod.encoder.graph_layer_1             0.000706      20039
2   _orig_mod.encoder.graph_layer_2             0.000612      20039
3       _orig_mod.encoder.decoder_1             0.000580      20039
4       _orig_mod.encoder.decoder_2             0.000566      20039
5          _orig_mod.encoder.linear             0.000300      20039
6   _orig_mod.encoder.vq.project_in             0.000532      20039
7  _orig_mod.encoder.vq.project_out             0.000625      20039
8    _orig_mod.encoder.vq._codebook             0.000632      20039
CPU Memory Usage: 1310.14 MB
Max GPU Memory Used: 1244.59 MB
